src/app/shared/form/builder/form-builder.service.ts

```diff
@@ -328,6 +389,82 @@ export class FormBuilderService extends DynamicFormService {
     return (tempModel.id !== tempModel.name) ? tempModel.name : tempModel.id;
   }
 
+  /**
+   * Add new form model to formModels map
+   * @param id id of model
+   * @param model model
+   */
+  addFormModel(id: string, model: DynamicFormControlModel[]): void {
```

Comment:
Same here, I cannot seem to find where most of the new methods in `form-builder.service.ts` are actually used.  Are these necessary for this PR, or maybe they were included by accident?

If they are necessary, it might be nice to see if we can find a way to add tests for them.  But, I'm currently not finding how/when they are used.